website/docs/tech-explanation/ckb-address.mdx

```diff
@@ -69,3 +70,18 @@ args to encode:          b39bbc0b3673c7d36450bc14cfcdad2d559c6c64
 ---
 full address generated:  ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqxwquc4
 ```
+
+## How CKB Addresses Reflect Ownership Rules (with Demo)
+
+An CKB addresses represents **ownership rules**, rather than an account. Each address is derived from a Lock Script that governs how the associated funds can be spent. Unlike blockchains with fixed script logic, this design gives users granular control over which code they trust to manage their assets.
+
+CKB provides two ways to reference Scripts:
+
+- **Type Hash (Upgradable):** References a script via a <Tooltip>Type_ID</Tooltip>. The script can be updated while preserving the address, offering convenience but requiring trust in the updater.
+- **Data Hash (Immutable):** References the exact script binary, freezing it at a specific version. This ensures stability and predictability, but requires manual updates for changes.
+
+This design allows users to decide between convenience and immutability, freeze scripts to specific versions, opt into updates on their own terms, or even run multiple script versions simultaneously. As a result, a single key can generate multiple valid addresses, each reflecting a different trust model.
+
+### Interactive Demo: Craft CKB Address
+
+Try the interactive demo [Craft CKB Address](https://craft-ckb-address.vercel.app) to see how one key combined with different Script references produce multiple addresses.
```

Comment:
should we add the image of https://github.com/nervosnetwork/docs.nervos.org/pull/699/files#diff-ce66d8630b08c1c6008c8201a60af4b0aa0db4dc5de3452d011367cc4eb8e4b3 here? otherwise the image is not used.